### DISCUSSION TITLE
Update dashboard widget data-colspan attribute on change

### DISCUSSION THREAD

**Discussion by membla** [craftcms/cms#10286] [Ideas] [answered]
In order to easily do targeted styling of dashboard widgets by number of columns they are set to inhibit, it would be great if either the data-colspan attribute (or an additional class) was automatically updated when it is edited using the colspan selection.

**Comment by brandonkelly** [comment, marked as answer]
Done for the next release (8d355e5f6986fb2cd27554b25707e0cd2b03f537).

**Reply by brandonkelly** [reply]
3.7.27 is out now with that change.
